storybookjs/storybook
# Scrollbar supports theming again

Issue: #6753

Core issue is #6753. @ndelangen's #6785 reimplements simplebar styles. This PR adds theming support back in.


## What I did
Add theming support back in

## How to test
- Change to dark theme and see white scroll bar
- Change to light theme and see dark scroll bar

![Screen Recording 2019-05-16 at 09 21 AM](https://user-images.githubusercontent.com/263385/57819695-2502ac80-77bc-11e9-9974-dcadf0c71df7.gif)
![Screen Recording 2019-05-16 at 09 20 AM](https://user-images.githubusercontent.com/263385/57819696-2502ac80-77bc-11e9-8513-1095955b3654.gif)
